espflash/src/cli/mod.rs

```diff
@@ -52,6 +52,9 @@ pub struct ConnectArgs {
     /// Serial port connected to target device
     #[arg(short = 'p', long, env = "ESPFLASH_PORT")]
     pub port: Option<String>,
+    /// Require confirmation before auto-connecting to a recognized device.
+    #[arg(short = 'i', long)]
+    pub interactive: bool,
```

Comment:
Changes LGTM! The only thing that I have some doubts is the name of the argument. Maybe `interactive` is fine, other options that came to my mind `confirm-device`, `confirm`, `select-device`. Any input on this?

Reply:
I don't have a strong opinion about the name, so I'm happy to implement whichever you prefer. I guess I like "confirm" better than "select," and maybe also `confirm-port` is a reasonable option :man_shrugging:

Reply:
Awesome, let's go for `confirm-prot`!

Reply:
Done!